File: ClientApp/src/app/models/vehiculo.ts
import { Propietario } from './propietario';
import { Conductor } from './conductor';

export class Vehiculo {
        id:number;
        placa:string;
        marca:string;
        modelo:string;
        fechaAfiliacion:Date;
        propietarioId:number;   
        conductorId:number;
}

File: Models/Vehiculo.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace ProyectoSegundo.Models {
    public class Vehiculo {
        [JsonProperty ("id")]
        public int id { get; set; }

        [JsonProperty ("placa")]
        public string Placa { get; set; }

        [JsonProperty ("marca")]
        public string Marca { get; set; }

        [JsonProperty ("modelo")]
        public string Modelo { get; set; }

        [JsonProperty ("fechaAfiliacion")]
        public DateTime FechaAfiliacion { get; set; }

        [JsonProperty ("propietarioId")]
        public int PropietarioId { get; set; }

        [JsonProperty ("conductorId")]
        public int ConductorId { get; set; }

    }
}